pkgs/top-level/all-packages.nix

```diff
@@ -21930,6 +21930,10 @@ in
 
   m4rie = callPackage ../development/libraries/science/math/m4rie { };
 
+  mkchromecast = callPackage ../applications/misc/mkchromecast {
```

Comment:
I'd change this to:
``` nix
mkchromecast = python3Packages.callPackage ../applications/misc/mkchromecast { };
```
So you don't have to pass each `python3Packages` attribute explicitly.